plugins/modules/podman_container.py

```diff
@@ -737,19 +737,27 @@
   network:
     description:
       - Set the Network mode for the container
-        * bridge create a network stack on the default bridge
-        * none no networking
-        * container:<name|id> reuse another container's network stack
-        * host use the podman host network stack.
-        * <network-name>|<network-id> connect to a user-defined network
-        * ns:<path> path to a network namespace to join
-        * slirp4netns use slirp4netns to create a user network stack.
-          This is the default for rootless containers
+      - I(bridge) - create a network stack on the default bridgeAHHH
+      - I(none) - no networking
+      - I(container:<name|id>) - reuse another container's network stack
+      - I(host) - use the podman host network stack.
+      - I(<network-name>|<network-id>) - connect to a user-defined network
+      - I(ns:<path>) - path to a network namespace to join
+      - I(slirp4netns) - use slirp4netns to create a user network stack.
     type: list
     elements: str
     aliases:
       - net
       - network_mode
+    default: slirp4netns
```

Comment:
We don't set the default, leaving for Podman to have its own, in v5 it's not `slirp4netns`  anyway.